contrib/hook.json

```diff
@@ -9,6 +9,6 @@
         }
     },
     "stages": [
-        "prestart"
+        "poststart"
```

Comment:
Why do we change from pre to post?

I guess because we need the containerid?

Reply:
At post-start phase container state already exists in runc so we can update it. At pre-start phase only the conmon process has been created.

Reply:
okay, thanks!